docs/devices/ZNCLDJ12LM.md

```diff
@@ -36,6 +36,29 @@ You can send a subset of options, all options that won't be specified will be re
 
 After changing `reverse_direction` you will need to fully open and fully close the curtain so the motor will re-detect edges. `reverse_direction` will get new state only after this recalibration.
 
+## Xiaomi ZNCLDJ12LM with battery
+
+| Picture | ![Xiaomi ZNCLDJ12LM with battery](../images/devices/ZNCLDJ12LM_w_battery.jpg) |
+
+If motor is used without battery it loses configuration when power down. After that you need to perform end stops calibration again. You can use the following automation for that:
```

Comment:
Can you also provide an example with raw MQTT commands? (as not everybody might be running Home Assistant)

Reply:
Done, please check. It it the expected format?